Python/sysmodule.c

```diff
@@ -1793,7 +1789,13 @@ sys__getframe_impl(PyObject *module, int depth)
                          "call stack is not deep enough");
         return NULL;
     }
-    return _Py_XNewRef((PyObject *)_PyFrame_GetFrameObject(frame));
+
+    PyObject *pyFrame = _Py_XNewRef((PyObject *)_PyFrame_GetFrameObject(frame));
```

Comment:
```suggestion
    PyObject *pyFrame = Py_XNewRef((PyObject *)_PyFrame_GetFrameObject(frame));
```

Reply:
Is there a reason this wasn't changed here when the public name was added? This line wasn't my change.

Reply:
Seems like an oversight `Py_XNewRef` is just an alias for `_Py_XNewRef` but without underscore looks better. Either way is fine.